README.md

```diff
@@ -299,13 +303,46 @@ laptop asset should be converted to a folder, and the license moved into it.
   Pass `git-command-args` as arguments to `git`, using the Onyo repository as
   the git repository. This is most valuable when used in conjunction with
   `ONYO_REPOSITORY_DIR`.
-
+- `onyo new [--non-interactive, -I] directory`:
+
+  Creates a new `asset` in `directory`. The command opens a dialog that asks for
+  the field names defined by the asset name scheme, and after creation opens the
+  new `asset` file with the editor.
+  After the editing is done, the new file will be checked for the validity of
+  it's YAML syntax.
+
+  - `--non-interactive` : Suppress opening of editor after file creation.
+- `onyo history [--non-interactive, -I] asset | directory`:
+
+  Show the history of a `directory` or `asset` file. The tools to show the
+  history are defined in `.onyo/config`.
```

Comment:
Redundant. Remove here, It's said better 2-3 sentences later.

Reply:
The sentence "The tools to show the  history are defined in `.onyo/config`" was removed.